### ISSUE TITLE
vendor/campaignmonitor/createsend-php/class/serialisation.php:38 Function utf8_encode() is deprecated

### ISSUE THREAD

**Issue by wesyah234** [campaignmonitor/createsend-php#119]
Trying with php 8.2, I received the above deprecation message.  Other than that, adding and deleting subscribers worked fine.

**Comment by markmorris** [comment]
Yeah i get this too, im admin currently stays in debug mode, so this is an issue for me. Ive even tried converting the strings to UTF-8 before i give them to CM, still errors.

**Comment by markdavies** [comment]
This is still an issue for me too ...

**Comment by cardi777** [comment]
issue for me too

**Comment by gedm** [comment]
Change line 38 of serialisation.php from
`$v = utf8_encode($v);`
to
`$v = mb_convert_encoding($v, 'UTF-8', 'ISO-8859-1');`

**Comment by wesyah234** [comment]
Thanks.  I just looked and there are 11 open pull requests on this repo, going back to 2016, the 2 most recent ones both providing the above fix for the serialisation.php file, submitted in June 2023 and November 2023.  Is it safe to say Campaign Monitor has abandoned this library?  I sure hope not!
